js/okx.js

```diff
@@ -3930,7 +3930,7 @@ module.exports = class okx extends Exchange {
         };
     }
 
-    async fetchTransfer (id, since = undefined, limit = undefined, params = {}) {
+    async fetchTransfer (id, code = undefined, params = {}) {
```

Comment:
We mist remove all references to `since` and `limit` from it, on line 3966.

Reply:
ah,  I thought with the combination of the second PR related to this issue, could be merged together, but right, i had to edit this PR too. done!